edgelet/edgelet-core/src/settings.rs

```diff
@@ -459,6 +464,62 @@ impl Listen {
     pub fn management_uri(&self) -> &Url {
         &self.management_uri
     }
+
+    pub fn min_tls_version(&self) -> Option<&Protocol> {
+        self.min_tls_version.as_ref()
+    }
+}
+
+#[derive(Clone, Debug, PartialEq)]
+pub enum Protocol {
+    Ssl3,
```

Comment:
Ssl3 is deprecated. I don't think we need it.

Reply:
IMO we have to support only the latest TLS version :)

Reply:
Yes.. backward compat is the only concern.